omod/src/main/java/org/openmrs/module/attachments/web/controller/AttachmentsController.java

```diff
@@ -35,6 +28,19 @@
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.multipart.MultipartHttpServletRequest;
 
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Iterator;
+
+import static org.openmrs.module.attachments.AttachmentsContext.getContentFamily;
+
+/*
+ * @Deprecated and replaced by
+ * {@link org.openmrs.module.attachments.rest.AttachmentBytesResource1_10 AttachmentBytesResource}
+ * and
+ * {@link org.openmrs.module.attachments.rest.AttachmentResource1_10 AttachmentResource}
+ */
+@Deprecated
```

Comment:
If possible you should also get rid of
```java
public static String getExtension(String fileName, String mimeType) { ...
```
and use the same one copied over to the resource controller.

Reply:
@chinzou did you see the above comment?

Reply:
yes, i was not able to use it since it's in **omod** subproject and **AttachmentBytesResource** is in **omod-1.10**